packages/webview_flutter/tizen/src/webview.cc

```diff
@@ -164,14 +165,20 @@ WebView::WebView(flutter::PluginRegistrar* registrar, int viewId,
       height_(height),
       working_surface_(nullptr),
       candidate_surface_(nullptr),
-      rendered_surface_(nullptr),
       is_mouse_lbutton_down_(false),
+      rendered_surface_(nullptr),
       has_navigation_delegate_(false),
       has_progress_tracking_(false),
       context_(nullptr),
       texture_variant_(nullptr),
       platform_window_(platform_window) {
-  tbm_pool_ = std::make_unique<BufferPool>(width, height);
+  use_sw_backend_ = needsSwBackend();
```

Comment:
```suggestion
  use_sw_backend_ = NeedsSwBackend();
```
Please change the 'n' to uppercase.

Reply:
Good point! done!